src/GdbCommand.h

```diff
@@ -27,28 +28,146 @@ class GdbCommand {
   virtual std::string invoke(GdbServer& gdb_server, Task* t,
                              const std::vector<std::string>& args) = 0;
 
+  /**
+   * When call, gdb will automatically fill argument to this command
+   * by running gdb.execute(). This is useful to gdb state along side
+   * of the command invocation.
```

Comment:
I'll split this out of the pull request too. I have to leave the auto_args stuff however since it's important to give good context for info checkpoint. That part is important to this PR.